style2.css

```diff
@@ -0,0 +1,112 @@
+body{
+background-color: cadetblue;
+font-size: 1em;
+width: 100%;
+}
+.imgsection{
+font-weight: 500;
+font-size: 1rem;
+line-height: 1;
+margin: 0 0 4em;
+
+}
+
+.project{
+  display: block;
+  width: 100%;
+  text-decoration: none;
+  margin: 1em 0;
+  cursor: pointer;
+}
+figure{
+  margin: 0;
+  padding: 0;  
+}
+.project figure img {
+  width: 100%;
+  height:300px;
+  display: block;
+  object-fit: cover;
+}
+.project figcaption{
+  padding: 2em;
+  width: 100%;
+  background: darkgreen;
+}
+.project h3{
+  color: white;
+}
+
+.cta{
+margin: -10 20px 0 0;
+display: flex;
+justify-content: flex-end ;
+}
+
+@media screen and (min-width: 960px ){
+  .imgwrapper{
+    max-width: 960px;
+    margin: 0 auto;
+  }
+  .imgcontainer{
+    display: grid;
+    grid-template-columns: repeat(3,1fr);
+    grid-gap: 1em;
+
+  }
+  .project:nth-child(1){
+    grid-column: 1/ span      3;
+  }
+  
+  .project{
+    width: 100%;
+    height: 300px;
+    margin: 0;
+
+  }
+  .project figure{
+    position: relative;
+    overflow: hidden;
+    margin-top: -10px;
+  }
+  .project figure img {
+    position: relative;
+    z-index: 999;
+    transition: transform 0.4s;
+
+  }
+  .project figure:hover img {
+    transform: translateY(-100px)
```

Comment:
missing semi-colon` ; `at the end of the line. might be messing with the rest of the code! add it!